src/autotrainer/autotrainer/autotrainer.py

```diff
@@ -23,12 +25,16 @@ def upload_multiple_images(self, container: Container, image_paths: [str], label
         labelled_blobs = []
         for path in image_paths:
             labelled_blobs.append(self.blob.add_data_from_path(container.value, path, labels, parent ))
+
         return labelled_blobs
 
+    def create_record_of_images(self, cvResults: [ImageCreateResult], container: Container):

```

Comment:
I think we need some test coverage of this line, and also another method to consume the records once they are in table_storage.